docs/guides/help-center.md

```diff
@@ -0,0 +1,63 @@
+---
+id: camunda-help-center
+title: Access the Camunda Help Center
+sidebar_label: Access the Camunda Help Center
+description: "Get recommendations for additional learning, step through the automation project guide, have a closer look at various use cases, access Camunda Academy, share feedback, and more."
+keywords: [getting-started, help-center, help, support]
+---
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--cloud">Camunda 8 SaaS only</span>
+<span class="badge badge--medium">Time estimate: 10 minutes</span>
+
+Camunda 8 SaaS offers a Help Center to all users, where you can access additional documentation, step through various use cases in Camunda, share your feedback, and more.
+
+## Visiting the Help Center
+
+You can visit the Help Center directly from your Camunda 8 account. From Console or Modeler, click the **Open Help Center** icon in the top right corner of the navigation:
+
+![question mark icon to open the help center](./img/open-help-center.png)
+
+This launches a modal for the Help Center, where you are taken to the **Your recommendations** tab.
+
+## New user survey
+
+At the top of **Your recommendations**, you may note an info box prompting you to complete the new user survey for a list of custom recommendations for additional learning.
+
+Click **Resume Survey** if you would like custom recommendations. Here, you may add additional details about your profile, goals, use case, and experience level. Fill out these details and click **Next**. Your recommendations will automatically update based on your responses.
+
+Click **Retake the survey** at any time for a new list of recommendations.
+
+## Using the Help Center
+
+### Your recommendations
+
+As mentioned, the **Your recommendations** tab populates a list of recommended resources and next steps based on your sign-up or in-app onboarding survey.
+
+These resources may link to additional material in the Console, documentation, Camunda Academy, or blog.
```

Comment:
```suggestion
These resources may link to additional material in the Console or Camunda's Academy, documentation, or blog.
```

Reply:
Is it not "Camunda Academy"?

Reply:
Or was this intended to change the ordering?